src/index.js

```diff
@@ -1,11 +1,83 @@
 import './style.css';
 import { Nav } from './components.js';
 import Helper from './helper.js';
+import Icon from './assets/images/download_120262.png';
 
 const helper = new Helper();
 
 document.querySelector('header').innerHTML = Nav();
 
 Helper.displayData('a');
 
-document.querySelector('form').addEventListener('submit', helper.searchHandler);
\ No newline at end of file
+document.querySelector('form').addEventListener('submit', helper.searchHandler);
+
+const icons = new Image();
+icons.src = Icon;
+
+setTimeout(() => {
```

Comment:
I am not sure why you are using a setTimeout function  but it is probably not best practice. Let's try to keep to best practices as much as we can